doc/classes/WorldEnvironment.xml

```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="WorldEnvironment" inherits="Node" category="Core" version="3.0-rc1">
 	<brief_description>
-		Sets environment properties for the entire scene
+		Sets environment properties for the entire scene (Post Processing Effects, Lightning and Background Settings)
 	</brief_description>
 	<description>
-		The [code]WorldEnvironment[/code] node can be added to a scene in order to set default [Environment] variables for the scene. The [code]WorldEnvironment[/code] can be overridden by an [Environment] node set on the current [Camera]. Additionally, only one [code]WorldEnvironment[/code] may be instanced in a given scene at a time. The [code]WorldEnvironment[/code] allows the user to specify default lighting parameters (e.g. ambient lighting), various post-processing effects (e.g. SSAO, DOF, Tonemapping), and how to draw the background (e.g. solid color, skybox).
+		The [code]WorldEnvironment[/code] node can be added to a scene in order to set default [Environment] variables for the scene. The [code]WorldEnvironment[/code] can be overridden by an [Environment] node set on the current [Camera]. Additionally, only one [code]WorldEnvironment[/code] may be instanced in a given scene at a time. The [code]WorldEnvironment[/code] allows the user to specify default lighting parameters (e.g. ambient lighting), various post-processing effects (e.g. SSAO, DOF, Tonemapping), and how to draw the background (e.g. solid color, skybox). Usually, these are added in order to improve the Realism/Color Balance of the Scene. Godot 3.0 supports Paranomas to be set as the background of a Scene. Without using Paranoma Image, the Sky color and Sun can be Placed in the Scene.
```

Comment:
Change ```Paranoma``` to ```Panorama```.

Reply:
And I don't think it's necessary write some letters with uppercase in the same sentence like this:
```the Sky color and Sun can be Placed in the Scene```

This way is ok:
```the sky color and sun can be placed in the scene```

Reply:
Same remarks as @guilhermefelipecgs: 
useless capital letters in `Realism/Color Balance` for example.